src/Symfony/Bundle/FrameworkBundle/Command/TranslationDebugCommand.php

```diff
@@ -40,6 +40,7 @@
  */
 class TranslationDebugCommand extends Command
 {
+    public const EXIT_CODE_SUCCESS = 0;
```

Comment:
This is defined on `Command`

https://github.com/symfony/symfony/blob/6.1/src/Symfony/Component/Console/Command/Command.php#L37

Reply:
Thanks! Fixed it.